srcdoc/configure/http1_directives.mt

```diff
@@ -15,6 +15,13 @@ $ctx->{directive}->(
     desc    => q{Timeout for incoming requests in seconds.},
 )->(sub {});
 
+$ctx->{directive}->(
+    name    => "http1-request-io-timeout",
+    levels  => [ qw(global) ],
+    default => 'http1-request-io-timeout: 5',
```

Comment:
This value was taken from: https://github.com/h2o/h2o/blob/f7138190/include/h2o.h#L85-L86